### DISCUSSION TITLE
Is there any XML parser available?

### DISCUSSION THREAD

**Discussion by vsfonseca23** [zepp-health/zeppos-forum#15] [Developer Feedback]
Hi guys,

Is there any XML parser available on the framework (device or side)? 

I've tried window.DOMParser and jQuery for npm, but none of these worked.

**Comment by MyoungXUE** [comment]
Thanks for your support to the forum.

Sorry, we don't support this feature at the moment. This is a very good idea and we will consider supporting it in a subsequent release.

**Reply by vsfonseca23** [reply]
Hi @MyoungXUE,

No problem, I'm just going to use some vanilla Javascript string functions to parse the XML data for now.

But a native XML parser should be a nice tool to have. 

In my scenario, I'm using the fetch API to fetch some data and sometimes it comes in XML format instead of JSON, generally from legacy APIs or when fetching HTML pages.
